Dockerfile

```diff
@@ -44,11 +88,20 @@ RUN mkdir -p $ANDROID_HOME; \
 # Install Node.JS
 RUN cd /home/jenkins && \
     wget --output-document=nodejs.tgz --quiet \
-      'https://nodejs.org/dist/v8.9.3/node-v8.9.3-linux-x64.tar.xz' && \
+      'https://repository.cliqz.com/dist/android/nodejs/node-v8.9.3-linux-x64.tar.xz' && \
     tar xf nodejs.tgz && \
     rm -f nodejs.tgz
 
-#Installation of 'yarn' for the Extension, 'appium' & 'wd' for Integration Tests
-RUN npm install -g yarn
-RUN npm install -g appium
-RUN npm install -g wd
\ No newline at end of file
+#Installation of 'yarn'; 'appium' & 'wd' for Integration Tests
+RUN npm install --global \
+    yarn \
+    appium \
+    wd
+
+#Install Ruby and Fastlane
+SHELL ["/bin/bash", "-l", "-c"]
```

Comment:
this one should probably go up, near ENV

Reply:
Makes sense. Will do it. 👍